scripts/set-git-source.sh

```diff
@@ -40,6 +40,7 @@ find ${SCRIPTDIR}/../0-bootstrap -name '*.yaml' -print0 |
       sed -i'.bak' -e "s#\${GIT_GITOPS_SERVICES_BRANCH}#${GIT_GITOPS_SERVICES_BRANCH}#" $File
       sed -i'.bak' -e "s#\${GIT_BASEURL}/\${GIT_ORG}/\${GIT_GITOPS_APPLICATIONS}#${GIT_BASEURL}/${GIT_ORG}/${GIT_GITOPS_APPLICATIONS}#" $File
       sed -i'.bak' -e "s#\${GIT_GITOPS_APPLICATIONS_BRANCH}#${GIT_GITOPS_APPLICATIONS_BRANCH}#" $File
+      sed -i'.bak' -e "s#\${GIT_BRANCH}#${GIT_BRANCH}#" $File
```

Comment:
Hi @phemankita thanks for catching this.

I think don't want this here, with the recent I change I updated this file but I forgot all the kustomizations.yaml for bootstrap

Its better to update the kustomization.yaml to use `GIT_GITOPS_BRANCH` instead of `GIT_BRANCH` like this one https://github.com/cloud-native-toolkit/multi-tenancy-gitops/blob/master/0-bootstrap/single-cluster/kustomization.yaml#L16

Reply:
Made the necessary changes. Thank you